packages/core/test/acceptance/view_insertion_spec.ts

```diff
@@ -541,4 +541,74 @@ describe('view insertion', () => {
     });
   });
 
+  describe('non-regression', () => {
+
+    // https://github.com/angular/angular/issues/33679
+    it('should insert views into ViewContainerRef injected by querying <ng-container>', () => {
+
+      @Component({
+        selector: 'app-root',
+        template: `
+        <div>container start|</div>
+        <ng-container #container></ng-container>
+        <div>|container end</div>
+
+        <ng-template #template >test</ng-template>
+        <div (click)="click()" >|click</div>
+        `
+      })
+      class AppComponent {
+        @ViewChild('container', {read: ViewContainerRef, static: true})
+        vcr !: ViewContainerRef;
+
+        @ViewChild('template', {read: TemplateRef, static: true}) template !: TemplateRef<any>;
+
+        click() { this.vcr.createEmbeddedView(this.template, undefined, 0); }
+      }
+
+      TestBed.configureTestingModule({
+        declarations: [AppComponent],
+      });
+      const fixture = TestBed.createComponent(AppComponent);
+      fixture.detectChanges();
+      expect(fixture.nativeElement.textContent).toBe('container start||container end|click');
+
+      fixture.componentInstance.click();
+      fixture.detectChanges();
+      expect(fixture.nativeElement.textContent).toBe('container start|test|container end|click');
+    });
+
+    it('should properly insert before views in a ViewContainerRRef injected on ng-container',
+       () => {
+         @Component({
+           selector: 'app-root',
+           template: `            
+          <ng-template #parameterListItem let-parameter="parameter">
+            {{parameter}}
+          </ng-template>
+          <ng-container *ngFor="let parameter of items;"
+            [ngTemplateOutlet]="parameterListItem"
+            [ngTemplateOutletContext]="{parameter:parameter}">
+          </ng-container>
+        `
+         })
+         class AppComponent {
+           items = [1];
+         }
+
+         TestBed.configureTestingModule({
+           declarations: [AppComponent],
+           imports: [CommonModule],
+         });
+
+         const fixture = TestBed.createComponent(AppComponent);
+         fixture.detectChanges();
+
```

Comment:
could you add intermediary assertion?
```suggestion

expect(fixture.nativeElement.textContent.trim()).toContain('1');
```

Reply:
Done